src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -397,15 +399,6 @@ private void hutDecClicked(@NotNull final Button button)
      */
     private List<String> getStyles()
     {
-        try
```

Comment:
@Raycoms You are right, 
I will load the client side directory but only when the worlds is not remote.
I guess I could do the same with the scan folder while I am at it ?